include/windows/config.h

```diff
@@ -192,3 +192,7 @@
 /* Version number of package */
 #define _FI_EXP(s) #s
 #define VERSION _FI_EXP(FI_MAJOR_VERSION) "." _FI_EXP(FI_MINOR_VERSION) ".0"
+
+#ifndef VERSION_SUFFIX
+#define VERSION_SUFFIX ""
```

Comment:
What's this for -- was it accidentally left over?

Reply:
Yes, it was accidentally left over here